platform/ios/Integration Tests/MGLTabBarControllerTests.m

```diff
@@ -0,0 +1,74 @@
+
+#import "MGLMapViewIntegrationTest.h"
+
+@interface MGLTabBarControllerTests : MGLMapViewIntegrationTest
+
+@property (nonatomic) UIViewController *viewController;
+@property (nonatomic) UITabBarController *tabController;
+@end
+
+@implementation MGLTabBarControllerTests
+
+- (void)setUp {
+    [super setUp];
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"IntegrationTest" bundle:[NSBundle mainBundle]];
+    [self.window setRootViewController:(UITabBarController *)[storyboard instantiateViewControllerWithIdentifier:@"TabController"]];
+    
+    // Access the tab controller from the story board, then access one of the childViewControllers.
+    self.tabController = (UITabBarController *)self.window.rootViewController;
+    self.viewController = self.tabController.childViewControllers[0];
+    
+    [self.viewController.view addSubview:self.mapView];
+    
+    MGLPointAnnotation *annot = [[MGLPointAnnotation alloc] init];
+    annot.coordinate = self.mapView.centerCoordinate;
+    [self.mapView addAnnotation:annot];
+}
+
+- (void)testViewControllerNil {
```

Comment:
Separate tests are completely new instances, so I think you could move this test into the main one below.